Portal/test/Datahub.SpecflowTests/Steps/DatahubAzureSubscriptionSteps.cs

```diff
@@ -69,7 +69,8 @@ public async Task WhenANewSubscriptionIsAdded()
         var subscription = new DatahubAzureSubscription()
         {
             SubscriptionId = Testing.WorkspaceSubscriptionGuid,
-            TenantId = Testing.WorkspaceTenantGuid
+            TenantId = Testing.WorkspaceTenantGuid,
```

Comment:
This is repeated all over the place, maybe there should be a helper method or two. CreateTestingSubscription, CreateInvalidSubscription.